ddtrace/internal/remoteconfig/client.py

```diff
@@ -325,8 +317,7 @@ def is_subscriber_running(self, pubsub_to_check):
     def reset_products(self):
```

Comment:
nit: do we also want return type here? Or are we only adding type annotations for the methods that previously had type comments

Reply:
A return type specification on this function would be ideal, but in this change I only converted existing type comments to annotations rather than adding new annotations.